tree/dataframe/inc/ROOT/RDF/RVariation.hxx

```diff
@@ -157,50 +162,28 @@ class R__CLING_PTRCHECK(off) RVariation final : public RVariationBase {
       auto &&results = fExpression(fValues[slot][S]->template Get<ColTypes>(entry)...);
       (void)entry; // avoid unused parameter warnings (gcc 12.1)
 
-      if (!ResultsSizeEq(results, fVariationNames.size(), fColNames.size())) {
+      if (!ResultsSizeEq(results, fVariationNames.size(), fColNames.size(), IsSingleColumn{})) {
```

Comment:
If you pass `bool` as template parameter, you can call this as `std::integral_constant<bool, IsSingleColumn>` (which is identical to `true_type` or `false_type`).

Reply:
ah, that's right. and you are saying that having a `bool` as a template parameter instead of a generic `typename` that by convention must be an `integral_constant<bool, ...>` is more robust. makes sense, will do.

Reply:
(done)